src/components/createItem.css

```diff
@@ -0,0 +1,4 @@
+/* .radio-buttons{
+    display:flex;
+    flex-direction: column;
+} */
```

Comment:
Do we still need this file?

Generally, there are different file structure systems in react. One way you can structure your files is to have component folders that have `jsx` and `css` files in them. 
For example: createItem(folder) has two files inside. 1. CreateItem.jsx 2. CreateItem.css. 
_This is just a recommendation_ 🙂

You can also view what [react team recommends](https://reactjs.org/docs/faq-structure.html).